test/Microsoft.TestPlatform.PerformanceTests/ProtocolV2Tests.cs

```diff
@@ -49,76 +52,79 @@ public class ProtocolV2Tests
     [TestMethod]
     public void TestCaseSerialize2()
     {
-        Serialize(TestCase, 2);
+        SerializeV2(TestCase);
         Stopwatch sw = Stopwatch.StartNew();
         for (int i = 0; i < 10000; i++)
         {
-            Serialize(TestCase, 2);
+            SerializeV2(TestCase);
         }
         sw.Stop();
 
-        VerifyPerformanceResult("TestCaseSerialize2", 2000, sw.ElapsedMilliseconds);
+        var actualDuration = sw.Elapsed;
+        actualDuration.Should().BeLessOrEqualTo(2.Seconds(), $"when serializing 10k test cases");
```

Comment:
Nit: interpolation is not needed.
```suggestion
        actualDuration.Should().BeLessOrEqualTo(2.Seconds(), "when serializing 10k test cases");
```